### PR TITLE
add test coverage to readme

### PR DESCRIPTION
**Pull request by mostafaibractility**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+
+N/A
+
+### Features
+
+N/A
+
+### Bugfixes
+
+Added code coverage to README
```
Comment (martintheault) [inline CHANGELOG.md:13]: It should be in Features

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
+## 0.1.1
```
Comment (martintheault) [inline CHANGELOG.md:1]: 0.2.0

File: examples/simple-driver/README.md [hunk 3]
```diff
@@ -426,3 +426,10 @@ To execute tests, you must use the following command:
 ```shell
 npm test
 ```
+### Test coverage
+
+To get the coverage of your tests, you must use the following command:
```
Comment (martintheault) [inline examples/simple-driver/README.md:431]: We must add that the important value in the report is `stmts` (statements)

**Review by martintheault — CHANGES_REQUESTED** [review]

File: examples/simple-driver/README.md (hunk 3 above)
Comment (mostafaibractility) [reply examples/simple-driver/README.md:431]: done

File: CHANGELOG.md (hunk 2 above)
Comment (mostafaibractility) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 1 above)
Comment (mostafaibractility) [reply CHANGELOG.md]: fixed

**Review by martintheault — APPROVED** [review]

**Pull request merged**